### ISSUE TITLE
ImportError when running plot scripts or solar radiation

### ISSUE THREAD

**Issue by reyery** [architecture-building-systems/CityEnergyAnalyst#2472]
This affects developers of CEA when they are switching to and from branches.

Related issues:  #2240
Usually the Exception raised is an `ImportError`

Problem
-
.pyc files from new .py files created in other branches will still remain even when the current branch does not have such .py files. 

This would sometimes cause some weird behavior while developing, for example, showing plots that do not exist in the current branch due to how it finds and list plots

https://github.com/architecture-building-systems/CityEnergyAnalyst/blob/f1cf5744830dc5f39376f8625b4bef301bfdffe4/cea/plots/categories.py#L87-L100

The current way to solve this issue is to use the _**Clean Python Complied files**_ function in PyCharm to delete these .pyc files, after switching to another branch. _(which works but sometimes we forget)_

Possible Solution
-
One possible solution is to use a **_git post-checkout hook_** which would run a script to remove .pyc files.
(as written here: https://codeinthehole.com/tips/a-useful-git-post-checkout-hook-for-python-repos/)

**Comment by JIMENOFONSECA** [comment]
@reyery let's do it. It is a very silent error we should avoid

**Comment by reyery** [comment]
I have tested the **git post-checkout hook** script and it works great on my machine. But after trying it a few more time using different method, I have found some problems with this method.

1. The git hook script only exists in my machine and it does not sync upstream to GitHub since it is located in the `.git` folder.
    - One way of solving this is to put the script in a folder named `.githooks` and have a script/installer to copy the hook into the `.git/hooks` folder.
    - An easier option is to set the directory of the hooks using [`git config core.hooksPath`](https://git-scm.com/docs/git-config#Documentation/git-config.txt-corehooksPath). However, it requires at least **git** version 2.9, but I don't think it would be a problem since I think we ship **git** with `cmder` in **Dependencies**.

2. The script linked above runs using **bash** i.e. `#!/usr/bin bash`. This wasn't a problem when I was using the cmd line (I think the included `cmder` bundles **bash**) but I soon realized that it does not work with **GitHub Desktop** (which I assume most of our developers are using) since it could not find any **bash** in the system.
   - We could point the script to use **bash** found in `cmder`.
   - Or we could use a **python** version of the script but we might still have to point it to a **python** executable path (which could be the one that we ship with CEA) if we assume that the user does not have **python** installed on their machine.

@daren-thomas any ideas?
